R/pkg/R/DataFrame.R

```diff
@@ -1271,12 +1280,8 @@ setMethod("[", signature(x = "DataFrame", i = "Column"),
 #'   subset(df, select = c(1,2))
 #' }
 setMethod("subset", signature(x = "DataFrame"),
-          function(x, subset, select, ...) {
-            if (missing(subset)) {
-              x[, select, ...]
-            } else {
-              x[subset, select, ...]
-            }
+          function(x, subset, select, drop=T, ...) {
```

Comment:
inconsistent with drop = FALSE in the signature of subset() in the R base package?